user_sync/app.py

```diff
@@ -367,32 +367,55 @@ def credentials():
     pass
 
 
-@credentials.command(
-    help="Stores all sensitive fields and updates configuration files, replacing plaintext values with keys")
+@credentials.command(help="Stores all sensitive fields and updates configuration files, replacing plaintext values with keys")
+
 def store():
     """
-    Stores credentials in the configuration file
+    Stores secure credentials in the configuration file
     This is an automated process
     """
+
     credential_manager = CredentialManager()
-    click.echo("Using backend: " + credential_manager.keyring_name)
-    click.echo("You have called the store-credential command.")
+    credential_manager.store()
+    retrieve_from_credman_keys = credential_manager.retrieve_from_credman_keys()
+    retrieve_config_credentials = credential_manager.retrieve_config_credentials()
+    if retrieve_config_credentials != retrieve_from_credman_keys:
```

Comment:
Let's let the credential manager worry about store errors - no need to check here